modules/home-environment.nix

```diff
@@ -592,10 +592,21 @@ in
         ''
       else
         ''
+          function hm-internal-replace-profile {
+
+            nix profile list \
+              | { grep 'home-manager-path$' || test $? = 1; } \
+              | awk -F ' ' '{ print $4 }' \
+              | cut -d ' ' -f 4 \
+              | xargs -t $DRY_RUN_CMD nix profile remove $VERBOSE_ARG
```

Comment:
`xargs`'s `-t` flag means echoing the command to stderr.
Because of this option, it prints out something like `nix profile remove /nix/store/***-home-manager-path` even when verbose flag is not specified. You don't want it to behave like this, do you?

Reply:
for now i just took the working code from the other part of the activation scripts, im happy to make it sanely silent

Reply:
IMAO you can just remove -t

Reply:
Oh well, just removing it still generates `removing 'home-manager-path'` message.
To make it completely silent you may need to redirect stdout to `/dev/null`.

Reply:
Oh, you just copied the existing code. Then this behaviour may be out-of-scope for this PR.

Reply:
@aiotter im happy to make the ux nice once its clear where i can put a test to get this from "works for me" to "works under test"